core-common/src/main/java/org/apache/kylin/common/KylinConfigBase.java

```diff
@@ -2709,7 +2709,11 @@ public int getDistCPMaxMapNum(){
         return Integer.valueOf(getOptional("kylin.storage.distcp-max-map-num", "50"));
     }
 
-    public String getKylinDictCacheStrength(){
+    public String getKylinDictCacheStrength() {
         return getOptional("kylin.dict.cache.strength", "soft");
-    };
+    }
+
+    public boolean encodeBaseCuboidColumnByColumn() {
```

Comment:
Could you please kindly add some explanation of this `property`? Maybe some suggestion about in which case should we enable this feature. Or explain it main principle.

Reply:
Sure, thanks you for your advice ~